README.md

```diff
@@ -2,51 +2,57 @@
 
 Staging: https://staging.x-hain.de [![status-badge](https://ci.x-hain.de/api/badges/xHain-hackspace/xhain-website/status.svg?branch=staging)](https://ci.x-hain.de/xHain-hackspace/xhain-website)
 
-Production: https://www.x-hain.de  [![status-badge](https://ci.x-hain.de/api/badges/xHain-hackspace/xhain-website/status.svg?branch=main)](https://ci.x-hain.de/xHain-hackspace/xhain-website)
+Production: https://www.x-hain.de [![status-badge](https://ci.x-hain.de/api/badges/xHain-hackspace/xhain-website/status.svg?branch=main)](https://ci.x-hain.de/xHain-hackspace/xhain-website)
 
 ## Getting started
 
 ### Install Hugo
 
 #### Mac OS
 
-* Install brew (for installing the rest): ``/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"``
-* Install go: ``brew install go``
-* Install hugo: ``brew install hugo``
+- Install brew (for installing the rest): `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- Install go: `brew install go`
+- Install hugo: `brew install hugo`
 
 #### For Windows/Linux
 
-* Install hugo: [Hugo Website](https://gohugo.io)
+- Install hugo: [Hugo Website](https://gohugo.io)
 
 ### Pull Submodules (eg. Theme)
 
-* Initial pull submodule xhain-theme: ``git submodule update --init --recursive``
-* Submodule update: ``git submodule foreach git pull origin master``
+- Initial pull submodule xhain-theme: `git submodule update --init --recursive`
+- Submodule update: `git submodule foreach git pull origin master`
+
+### Install Dependencies
+
+We use NPM to manage dependencies. Install these by running:
+
+`npm install`
 
 ### Generate Site
 
-* Run hugo to generate HTML: ``hugo``
-* Run hugo for local development:
+- Run hugo to generate HTML: `hugo`
+- Run hugo for local development:
 
-   ``hugo server --config config.toml,home.toml``
+    `hugo server --config config.toml,home.toml`
 
-  or
+    or
 
-  ``hugo server --config config.toml,home.toml -w --cleanDestinationDir``
+    `npm run dev`
 
 ## Content Editing
 
 ### New Post
 
-* create new post (using archetypes):
+- create new post (using archetypes):
 
-  ``hugo new de/post/YYYY-MM-DD_title.de.md -k post``
+    `hugo new de/post/YYYY-MM-DD_title.de.md -k post`
 
-  or
+    or
 
-  ``hugo new en/post/YYYY-MM-DD_title.en.md -k post``
+    `hugo new en/post/YYYY-MM-DD_title.en.md -k post`
 
-* just edit new post in the "content"-folder
+- just edit new post in the "content"-folder
 
 ## Calendar
 
```

Comment:
The formatting changes were made by prettier to bring the file in line with best practices